packages/bodiless-components/src/List/asBodilessList.tsx

```diff
@@ -50,8 +51,8 @@ type SubListComponents = {
 };
 
 const startComponents: SubListComponents = {
-  WrapperItem: asComponent('li'),
-  List: asComponent('ul'),
+  WrapperItem: Li,
```

Comment:
Sorry - I didn't mean to add these to the List API - I mean to add them to [asMenuSubList](https://github.com/wodenx/Bodiless-JS/blob/daacd0742f82923ef384f292281a79287c80a38b/packages/bodiless-organisms/src/components/Menu/SimpleMenu.tsx#L35-L39).  So far we are keeping components in @bodiless/components free of a dependency on the stylable part of fClasses (though of course they do depend on the design API, and maybe at some point we should abandon this policy).